paymentapi/src/main/resources/application.properties

```diff
@@ -46,3 +46,5 @@ paytrail_callback_delay=60
 payment.card_token.encryption.password=
 
 spring.devtools.restart.enabled=false
+
+payment.experience.url=http://payment-experience-api:8080/v1/payment/
```

Comment:
Pitäiskö tässä jättää tuo oletusarvo tyhjäksi ja ennemminkin laittaa defaultti arvo sille muuttujalle joka tätä käyttää. Nuo vaikuttaa olevan niin kovia koodauksia ettei tahdo muualta päivittyä

Reply:
Samalla tyylillä on tehty aiemmat order.experience.url:it. Mutta jos tämä "uusi" tyyli on mitä halutaan, niin muutetaan.

Reply:
Joo, noita ei yleensä ole varmaan tarvinnut muualta saada laitettua mutta käytännössä kaikki nuo propertiesit jotka halutaan asettaa toisaalla on tyhjinä application.properties filessä